app/src/main/java/fr/neamar/kiss/result/MyClass.java

```diff
@@ -0,0 +1,5 @@
+package fr.neamar.kiss.result;
```

Comment:
Is this useful?

Reply:
Nope, added in error, that must have happened when I tried to create a new app named `-`, I must have clicked on "new class" by mistake, and android studio is not helping by adding it to git by default.

Reply:
Then I didn't catch it via `git add -p`

Reply:
Arf.

Auto-add in Android Studio is something that I disabled after doing exactly this a couple times too ><